packages/nextjs/services/database/bg-imports/20250423_location_socials.ts

```diff
@@ -0,0 +1,101 @@
+import { db } from "../config/postgresClient";
+import { lower, users as schemaUsers } from "../config/schema";
+import * as dotenv from "dotenv";
+import { eq } from "drizzle-orm";
+import * as path from "path";
+
+dotenv.config({ path: path.resolve(__dirname, "../../../.env.development") });
+
+const BG_USERS_ENDPOINT = `${process.env.NEXT_PUBLIC_BG_BACKEND}/builders`;
+
+export type BGBuildersApiResponse = {
```

Comment:
Would it make sense to have a bg types file somewhere? (maybe inside bg-imports? and move the batch data stuff too) 

Since we will be using it in different places.

Reply:
Ohh yes moved it inside `types.ts` file in `bg-imports` at https://github.com/BuidlGuidl/SpeedRunEthereum-v2/pull/148/commits/d4adecc9ef25894468eca222cfa5d6d74dc17969